### PR TITLE
Update OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by duglin**
Fix-up KTC members

### CODE REVIEW

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,8 +1,8 @@
 aliases:
   knative-trademark-committee:
-    - bsnchan
-    - duglin
-    - ronavn
+    - evankanderson
+    - smoser
```
Comment (evankanderson) [inline OWNERS_ALIASES:4]: This needs to be `smoser-ibm`

**Comment by duglin** [comment]
Closing in favor of: https://github.com/knative/specs/pull/90
Don't ask....

**Pull request closed without merging**